srcpkgs/libnvidia-container/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'libnvidia-container'
+pkgname=libnvidia-container
+version=1.9.0
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz curl bmake groff m4 pkg-config which go"
+makedepends="rpcsvc-proto libcap-devel libseccomp-devel libtirpc-devel"
+short_desc="NVIDIA container runtime library"
+maintainer="Quentin Freimanis <quentinfreimanis@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/NVIDIA/libnvidia-container"
+distfiles="https://github.com/NVIDIA/libnvidia-container/archive/v${version}.tar.gz"
+checksum=5eb7c6de853307970a53e9f65cc61be7c0ceae315c175fb0a2271c5669dbf33f
+
+do_build() {
+	make dist prefix=/usr REVISION=${version}
```

Comment:
`prefix` is patched to `/usr` in `fix_makefile.patch`, is it necessary here too?

Reply:
removed the prefix patch and now only setting it in the template, thanks